test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -583,6 +583,7 @@ def test_pytorch_training_torchaudio_cpu(
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_TORCHAUDIO_CMD)
```

Comment:
we might want to add a new test to ensure our packages like pytorch, torchvision, torchaudio, torchtriton, etc are not being replaced